### ISSUE TITLE
Login with Github redirects to serviceWorker.js

### ISSUE THREAD

**Issue by alextrastero** [forem/forem#14598]
**After logging in with Github I was redirected to a static file `serviceWorker.js`**

Logged in with Github and was redirected to this page: https://dev.to/serviceworker.js?signin=true

Screenshot:
![image](https://user-images.githubusercontent.com/4152819/130954919-839a06e2-9fdd-486d-81cb-b7c4edfa9d46.png)

Chrome: Version 92.0.4515.159 (Official Build) (x86_64)
OS: MacOs

**Comment by citizen428** [comment]
Hmm, the service worker should no longer exist, see https://github.com/forem/forem/pull/12974

**Comment by alextrastero** [comment]
https://dev.to/serviceworker.js?signin=true

**Comment by citizen428** [comment]
Yes, this is the self-destructing service worker mentioned in the PR linked above. During that PR we also removed a redirect check from registrations controller, maybe that's the cause of the issue. cc @nickytonline

**Comment by nickytonline** [comment]
I'm not really sure how to reproduce this. I could probably go back to a local version of Forem, pre-removal of the service worker (SW), log in to activate the old SW, then grab the latest source code to have it self-destruct and see if this behaviour happens, but when I tested the self-destructing SW when we removed our SW implementation, I was using my GitHub account locally. I'm wondering if a hard refresh fixes this for you @alextrastero?

**Comment by mikeydorje** [comment]
Code review suggests this may be resolved or no longer applicable due to changes in this area since the issue was filed. Closing due to age and low activity. Please reopen with current repro steps if this persists.
